daprdocs/content/en/reference/components-reference/supported-state-stores/setup-mongodb.md

```diff
@@ -56,15 +58,18 @@ If you wish to use MongoDB as an actor store, append the following to the yaml.
 
 | Field              | Required | Details | Example |
 |--------------------|:--------:|---------|---------|
-| host               | Y        | The host to connect to | `"mongo-mongodb.default.svc.cluster.local:27017"`
-| username           | N        | The username of the user to connect with | `"admin"`
-| password           | N        | The password of the user | `"password"`
+| server             | Y<sup>1</sup> | The server to connect to, when using DNS SRV record | `"server.example.com"`
+| host               | Y<sup>1</sup> | The host to connect to | `"mongo-mongodb.default.svc.cluster.local:27017"`
+| username           | N        | The username of the user to connect with (applicable in conjunction with `host`) | `"admin"`
+| password           | N        | The password of the user (applicable in conjunction with `host`) | `"password"`
 | databaseName       | N        | The name of the database to use. Defaults to `"daprStore"` | `"daprStore"`
 | collectionName     | N        | The name of the collection to use. Defaults to `"daprCollection"` | `"daprCollection"`
 | writeconcern       | N        | The write concern to use | `"majority"`
 | readconcern        | N        | The read concern to use  | `"majority"`, `"local"`,`"available"`, `"linearizable"`, `"snapshot"`
 | operationTimeout   | N        | The timeout for the operation. Defautls to `"5s"` | `"5s"`
 
+> [ <sup>1</sup> ] The `server` and `host` fields are mutually exclusive. If neither or both are set, Dapr will return an error.
```

Comment:
The [<sup>1</sup>] looks a little weird to me. Should we include the [] in the sup: <sup>[1]</sup>?